experimental/datatest/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "datatest"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+icu_codepointtrie = { version = "0.3", path = "../../utils/codepointtrie", features = ["serde"] }
+icu_provider = { version = "0.4", path = "../../provider/core", features = ["macros"] }
+serde = { version = "1.0", features = ["derive", "alloc"]}
+zerovec = { version = "0.4", path = "../../utils/zerovec", features = ["serde"] }
+toml = "0.5"
+postcard = { version = "0.7", features = ["use-std"] }
+
+[features]
+default-features = ["provider_serde"]
+provider_serde = []
```

Comment:
You don't have a `provider_serde` feature in code, remove?

Reply:
Removed this test crate.